service/src/main/java/bio/terra/workspace/service/resource/referenced/ReferencedGitRepoResource.java

```diff
@@ -0,0 +1,39 @@
+package bio.terra.workspace.service.resource.referenced;
```

Comment:
I don't think this file should be in this PR - did it get pulled in from a different branch you're working on?